.ci/validation.sh

```diff
@@ -655,7 +655,7 @@ no-error-orekit)
   checkout_from https://github.com/Hipparchus-Math/hipparchus.git
   cd .ci-temp/hipparchus
   # checkout to version that Orekit expects
-  SHA_HIPPARCHUS="815ad2bf9ce764e4498911d2145c49165f5f3333"
+  SHA_HIPPARCHUS="1492f06848f57e46bef911aad16203a242080028"
```

Comment:
`SHA_HIPPARCHUS="1492f06848f57e46bef911a""ad16203a242080028"`

it will help to avoid extension on whitelist, that we try to keep clean.
please  apply same trick for other SHA